### ISSUE TITLE
rsnapshot: can't override the src (/ → /var/backups/backup2l)

### ISSUE THREAD

**Issue by mathieumd** [debops/debops#2069]
I want to backup only the remote `/var/backups/backup2l` directory (no need to backup all `/` again, local Backup2l already does it for me).

In [`rsnapshot/defaults/main.yml` L852](https://github.com/debops/debops/blob/master/ansible/roles/rsnapshot/defaults/main.yml#L852) the comment says *You can use the 'raw' parameter to override it.*

I'm not sure how this is supposed to be set up, while keeping the auto-replacement of `root@fqdn` by `root@test.example.com`!

In short, I would like this change in `/etc/rsnapshot/hosts/<host>/rsnapshot.conf`:
```diff
-backup  root@test.example.com:/      ./      +ssh_args=-i ~/.ssh/id_rsnapshot
+backup  root@test.example.com:/var/backups/backup2l      ./      +ssh_args=-i ~/.ssh/id_rsnapshot
```

(I tried to achieve the same result with excludes and includes, but with no success)
